.github/workflows/pr-label-milestone-check.yml

```diff
@@ -0,0 +1,27 @@
+name: PR Label and Milestone Check
+
+on:
+  pull_request:
+    types: [opened, edited, labeled, unlabeled, synchronize]
+
+jobs:
+  check_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
Should be using pinned action here

`uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7`